src/ProtocolGateway.Core/Mqtt/MqttAdapter.cs

```diff
@@ -800,11 +804,13 @@ bool TryMatchSubscription(string topicName, DateTime messageTime, out QualityOfS
                     return;
                 }
 
+                CommonEventSource.Log.Info("ClientAuthenticated", this.identity.ToString(), this.ChannelId);
```

Comment:
``` IDeviceIndentity ``` implementations do not override the ```ToString() ``` method. You will only log the fully qualified class name here. Either implement `` ToString() `` or change this to ``` this.identity.Id ```